.drone.star

```diff
@@ -0,0 +1,150 @@
+def main(ctx):
+    platforms = [ "amd64","arm64"]
+
+    volumes = [
+        {
+            "name": "cache",
+            "path": "/go",
+        }]
+
+    workspace = {
+        "path": "/go/src/github.com/${DRONE_REPO}"
+    }
```

Comment:
workspace is unused, it needs to be added to the pipelines.
Although I'm not sure what this does exactly, since we already have the `volumes` cache

Reply:
This was an omission and I have added it to all the pipelines.